apps/qubit/modules/informationobject/actions/fullWidthTreeViewAction.class.php

```diff
@@ -46,7 +46,7 @@ public function execute($request)
 
     // Impose limit to what nodeLimit parameter can be set to
     $maxItemsPerPage = sfConfig::get('app_treeview_items_per_page_max', 10000);
-    if (!ctype_digit($request->nodeLimit) || $request->nodeLimit > $maxItemsPerPage)
+    if (!intval($request->nodeLimit) || $request->nodeLimit > $maxItemsPerPage)
```

Comment:
Hi @mcantelon! Should nodeLimit be screened so it just does not permit values <= zero?

Reply:
Ah good call... fixed!